mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -416,37 +499,45 @@ def relevance_to_query():
                 "outputs": "The capital of France is Paris.",
             }
         ]
-        result = mlflow.genai.evaluate(data=data, scorers=[relevance_to_query()])
+        result = mlflow.genai.evaluate(data=data, scorers=[relevance_to_query])
     """
-    return _RelevanceToQuery()
-
 
-class _Safety(_BaseBuiltInScorer):
-    name: str = "safety"
+    name: str = "relevance_to_query"
     required_columns: set[str] = {"inputs", "outputs"}
 
     def __call__(self, *, inputs: Any, outputs: Any) -> Assessment:
-        """Evaluate safety of the response."""
+        """Evaluate relevance to the user's query."""
         return super().__call__(inputs=inputs, outputs=outputs)
 
+    def with_config(self, *, name: str = "relevance_to_query") -> "RelevanceToQuery":
+        """
+        Get a new scorer instance with a specified name.
+
+        Args:
+            name: The name of the scorer. Default is "relevance_to_query".
+        """
+        return RelevanceToQuery(name=name)
+
 
-@_builtin_scorer
 @experimental
-def safety():
+class Safety(_BaseBuiltInScorer):
     """
     Safety ensures that the agent's responses do not contain harmful, offensive, or toxic content.
 
     You can invoke the scorer directly with a single input for testing, or pass it to
     `mlflow.genai.evaluate` for running full evaluation on a dataset.
 
+    Use `mlflow.genai.scorers.safety` to get an instance of this scorer with
```

Comment:
Same comment as earlier - with each of the built-in evaluators and their compatibility with using `.with_config()`, can a consistent example be used within the API docs? .e.g you have the global guidelines docstring examples show effective usage of the new API, but not in all of the built-in scorers.